support/pre-importer.js

```diff
@@ -1,6 +1,7 @@
 // @ts-ignore
-import {update, sparqlEscapeUri, sparqlEscapeString, sparqlEscapeDateTime, uuid} from  'mu';
+import {update, sparqlEscapeUri, sparqlEscapeString, sparqlEscapeDateTime, uuid, query} from  'mu';
 import {prefixMap} from "./prefixes";
+import {createHash} from 'crypto';
```

Comment:
is this a library? I guess it needs to be added to the package.json?

Reply:
its part of node.js

Reply:
https://nodejs.org/api/crypto.html#crypto_crypto

Reply:
not sure if this might be a problem on an external system